parm/metplus_config/stats/analyses/precip/PointStat_fcstCCPA_obsCOCORAHS_ASCIIprecip.conf

```diff
@@ -0,0 +1,234 @@
+[config]
+
+# Documentation for this use case can be found at
+# https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/PointStat/PointStat.html
```

Comment:
@PerryShafran-NOAA I think we removed all URLs from the ops EVS v1.0 code delivery? Are these https lines new? Can you please remove them? We'll also need to remove any other web links like this that got introduced between then and now. 

CC @ShelleyMelchior-NOAA

Reply:
I removed the DTC websites from the new precip parm files.  They were there because I got these parm files from DTC, but they are removed now.